qiskit/visualization/matplotlib.py

```diff
@@ -583,14 +581,12 @@ def _multiqubit_gate(self, xy, fc=None, ec=None, gt=None, sc=None, text="", subt
                     wrap=True,
                 )
 
-    def _gate(self, xy, fc=None, ec=None, gt=None, sc=None, text="", subtext=""):
+    def _gate(self, op, xy, fc=None, ec=None, gt=None, sc=None, text="", subtext=""):
```

Comment:
when adding parameter to an existing function, it is common to add them at the end to avoid accidental wrongly binding from existing code:
```suggestion
    def _gate(self, xy, op, fc=None, ec=None, gt=None, sc=None, text="", subtext=""):
```
I dont think is a big issue in this case.